onnx_neural_compressor/algorithms/weight_only/rtn.py

```diff
@@ -99,21 +103,46 @@ def rtn_quantize(
 
             weight = quant_utils.pad_tensor(weight, group_size, k_blocks)
 
-            satisfy_MatMulNBits_condition = (
-                version.Version(ort.__version__) > constants.ONNXRT1161_VERSION and num_bits == 4
-            )
+            satisfy_MatMulNBits_condition = ort_version > constants.ONNXRT1161_VERSION and num_bits == 4
             satisfy_MatMulFpQ4_condition = (
-                version.Version(ort.__version__) >= constants.ONNXRT116_VERSION and num_bits == 4 and group_size == 32
+                ort_version >= constants.ONNXRT116_VERSION and num_bits == 4 and group_size == 32
```

Comment:
similar comments, it is better to put such a ORT version check for both MatMulNBits and standard DQ out of algorithm specific logic/file. maybe in high level API layer by detecting quant_format